doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+corresponding image pixel.
+
+As expected, this is a 768x1024 matrix:
+
+::
+
+    >>> img[:, :, 0].shape
+    (768, 1024)
+
+Since we are going to perform linear algebra on this data, it might be more
+interesting to have real numbers between 0 and 1 in each entry of the matrices
+to represent the RGB values. We can do that by setting
+
+    >>> img_array = img/255
+
+You can check if this worked by doing some tests; for example, inquiring about
+maximum and minimum values for this array:
+
+::
+
+    >>> img_array.max(), img_array.min()
+    (1.0, 0.0)
+
+or checking the type of data in the array:
+
+::
+
+    >>> img_array.dtype
+    dtype('float64')
+
+Linear algebra on an axis
+-------------------------
+
+It is possible to use methods from linear algebra to approximate an existing set
+of data. Here, we will use the SVD (Singular Value Decomposition) to try to
+extract the most important information from this image.
+
+Keep in mind that most linear algebra operations (functions in the
+`numpy.linalg` module) can also be found in `scipy.linalg`. For more information
+on this, check the `numpy.linalg` reference.
+
+To proceed, import the linear algebra submodule from either NumPy or SciPy:
+
+    >>> import numpy.linalg as LA
+
+To simplify, let's assign each color channel to a separate matrix:
+
+::
+
+    >>> red_array = img_array[:,:,0]
+    >>> green_array = img_array[:,:,1]
+    >>> blue_array = img_array[:,:,2]
+
+In order to extract information from each matrix, we can use the SVD to obtain
+3 arrays representing a decomposition. From the theory of linear algebra, given
+a matrix :math:`A`, the following decomposition can be computed:
+
+.. math::
+
+   U \Sigma V^T = A
+
+where :math:`U` and :math:`V^T` are square and :math:`\Sigma` is the same size
+as :math:`A` (so that the multiplication between :math:`U`, :math:`\Sigma` and
+:math:`V^T` works). :math:`\Sigma` is a diagonal matrix and contains the
+`singular values` of :math:`A`, organized from highest to smallest. These values
+are always non-negative, and can be used as an indicator of the "importance" of
+some features represented by this data matrix.
+
+Let's see how this works in practice with ``blue_array``.
+
+::
+
+    >>> U_blue, s_blue, Vt_blue = LA.svd(blue_array)
+
+.. note::
+
+    This command might take a while to run, depending on your hardware; don't
+    worry, as this is expected! The SVD is a pretty intensive computation.
+
+Let's check that this is what we expected:
+
+::
+
+    >>> U_blue.shape, s_blue.shape, Vt_blue.shape
+    ((768, 768), (768,), (1024, 1024))
+
+Note that ``s_blue`` has a particular shape: it has only one dimension. This
+means that some functions might work differently than expected for this array.
+For example, from the theory one might expect ``s_blue`` and ``Vt_blue`` to be
+compatible for multiplication. However, this is not true as ``s_blue`` does not
+have a second axis. Executing
+
+::
+
+    >>> s_blue @ Vt_blue
```

Comment:
worth noting that `@` is matrix multiplication, I think it's the first usage here